front/app/services/auth.ts

```diff
@@ -115,12 +114,7 @@ export function signOut() {
   }
 }
 
-export function signOutAndDeleteAccountPart1() {
-  setTimeout(() => eventEmitter.emit('tryAndDeleteProfile'), 500);
```

Comment:
This setTimeout caused me to debug for half a day 🙃 The user deletion got into an infinite loop of delete calls when the sign up modal got closed after my changes.

Great illustration that every line of code matters; and this was not adding anything unless I'm not seeing something.

Reply:
Timeouts can sometimes cause a lot of problems, indeed. Good that this one is gone 👍